lib/questionnaires/view/src/questionnaire_theme.dart

```diff
@@ -35,7 +35,7 @@ abstract class QuestionnaireTheme {
   bool showNullCodingOption();
 
   /// Returns whether each question will be preceded by its own ID.
-  bool showAnswerIndexOption();
+  bool showQuestionIndexOption();
```

Comment:
why is the parameter "showQuestionNumbers" and this is "showQuestionIndexOption"? I like "showQuestionNumbers" better.

Reply:
Me too...and that's what the user sees. I'm not a fan of creating/naming two variables for the same thing here.